src/microbe_stage/systems/MicrobeReproductionSystem.cs

```diff
@@ -89,8 +89,7 @@ public static (float RemainingAllowedCompoundUse, float RemainingFreeCompounds)
             float delta)
     {
         // Skip some computations when they are not needed
-        if (!worldSettings.PassiveGainOfReproductionCompounds &&
-            !worldSettings.LimitReproductionCompoundUseSpeed)
+        if (!worldSettings.LimitReproductionCompoundUseSpeed)
```

Comment:
Did I misread the flow of the code, or is there a problem here when the limiting is turned off? Because to my eyes it looks like turning off limiting will remove all passive growth. I think the right thing to do would be to remove this block entirely, and then rely on the 
```
        if (worldSettings.LimitReproductionCompoundUseSpeed)
        {
            remainingAllowedCompoundUse = remainingFreeCompounds * Constants.MICROBE_REPRODUCTION_MAX_COMPOUND_USE;
        }
```

code block to handle the limiting correctly.

Reply:
I think this comment wasn't yet resolved?